### PR TITLE
remaining renames from ServiceInstanceCredential to ServiceBinding

### PR DESCRIPTION
**Pull request by vaikas-google**
Final follow up for ServiceInstanceCredential to ServiceBinding renames.
Addresses: #1310

### CODE REVIEW

**Comment by pmorie** [comment]
Missed a link:

```console
Running href checker:
./docs/walkthrough-1.6.md: Can't find: ./docs/../contrib/examples/walkthrough/ups-instance-credential.yaml
```

File: docs/walkthrough-1.7.md [hunk 1]
```diff
@@ -277,18 +277,18 @@ see a new one:
 kubectl get secrets -n test-ns
 NAME                              TYPE                                  DATA      AGE
 default-token-3k61z               kubernetes.io/service-account-token   3         29m
-ups-instance-credential           Opaque                                2         1m
+ups-binding           Opaque                                2         1m
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: The spacing here got messed up.

Comment (vaikas-google) [reply docs/walkthrough-1.7.md]: Oopsies :)

**Comment by vaikas-google** [comment]
@pmorie Yeah, since that was marked deprecated, I didn't touch any of that. Does that even work anymore and are we testing against 1.6? I'll change it, not arguing against that, just curious if it even works or should it be removed.

**Comment by pmorie** [comment]
Let's make the link work for now and remove the 1.6 version in another PR

**Comment by vaikas-google** [comment]
Updated 1.6 with the correct resource names and all to match what is in 1.7.

**Pull request merged**